tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rsyslog_log_utils.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+RSYSLOG_CONF='/etc/rsyslog.conf'
+LOG_FILE_PREFIX=test
+RSYSLOG_TEST_DIR=/tmp
```

Comment:
You are right, it is defined as `mktemp -d` in create_rsyslog_test_logs sub. However, as this code is shared I considered it would be reasonable to have some default value in case this variable referenced outside of create_rsyslog_test_logs call (this is not the case for the current set of tests under review). Thoughts?

Reply:
Okay, we can have it here, you don't have to remove it.
I don't think that someone will use it outside of `create_rsyslog_test_logs`, but if, then we know it will work.